rpcs3/Emu/Cell/Modules/cellGem.cpp

```diff
@@ -804,7 +959,7 @@ s32 cellGemSetRumble(u32 gem_num, u8 rumble)
 		return CELL_GEM_ERROR_UNINITIALIZED;
 	}
 
-	if (!check_gem_num(gem_num))
+	if (!check_gem_num(gem_num) || rumble > 255)
```

Comment:
`rumble` is a `u8` so this is impossible.
libgem doesn't check anything regarding `rumble`.

Reply:
upsi. i just thought more than 255 is baka so I added that one without thinking